pykern/pkunit.py

```diff
@@ -555,6 +560,30 @@ def _failed_msg(process):
 """
             )
 
+        def _ndiff_config(epsilon, work_d):
+            pykern.pkio.write_text(
+                work_d.join(_NDIFF_CONF_FILE), f"* * abs={float(epsilon)}"
+            )
+
+        def _ndiff_files(expect_path, actual_path, epsilon):
+            _ndiff_config(epsilon, w)
+            p = subprocess.run(
+                (
+                    "ndiff",
+                    actual_path,
+                    expect_path,
+                    w.join(_NDIFF_CONF_FILE),
```

Comment:
This is what I call implicit coupling. Return the file path from _ndiff_config so that you don't have the implicit coupling of work_dir and NDIFF_CONF_FILE. Just call it in place (no temp var) and remove `_NDIFF_CONF_FILE` constant, because it is unnecessary.